module2/lessons/index.md

```diff
@@ -4,45 +4,55 @@ title: Module 2 - Lessons
-* [SQL and Active Record](./sql_and_active_record)
-* [Joins](./joins)
+
+- [Intro to Databases and ORMs](./databases_and_orms)
+- [SQL and Active Record](./sql_and_active_record)
+- [Joins](./joins)
 <!-- * [Joining Multiple Tables](./joins_2)-->
 
 ## HTML and CSS
-* [HTML: Fundamentals](./html_fundamentals)
-* [CSS: Fundamentals](./css_fundamentals)
-* [CSS: Flexbox](./css_flexbox)
-* [CSS: Intro To Layout](./css_intro_to_layout) (prework for Flexbox lesson)
+
+- [HTML: Fundamentals](./html_fundamentals)
+- [CSS: Fundamentals](./css_fundamentals)
+- [CSS: Flexbox](./css_flexbox)
+- [CSS: Intro To Layout](./css_intro_to_layout) (prework for Flexbox lesson)
 
 ## JavaScript
-* [JS: Fundamentals](./js_fundamentals)
-* [JS: Intro to the DOM](./js_intro_to_dom)
-* [JS: Event Listeners](./js_event_listeners)
-* [JS: Array Prototype Methods](./js_array_prototype_methods)
+
+- [JS: Fundamentals](./js_fundamentals)
+- [JS: Intro to the DOM](./js_intro_to_dom)
+- [JS: Event Listeners](./js_event_listeners)
+- [JS: Array Prototype Methods](./js_array_prototype_methods)
 
 ## Active Record
-* [Class vs Instance Methods](./class_vs_instance_methods.md)
+
+- [Class vs Instance Methods](./class_vs_instance_methods)
 
 ## Professional Development
-* [Overview](./pd_overview)
-* [Building Your Brand on LinkedIn](https://docs.google.com/presentation/d/e/2PACX-1vS8CZUjYkpnkTJrp2Ga8um-UUFUsJ-5JA85FF9x875J-l_eKy7IyL47sAt8kl_FOlg7rG5ntVxpk5he/pub?start=false&loop=false&delayms=3000)
-* [Networking, Outreach and Coffee Chats](./networking)
-* [Networking](https://docs.google.com/presentation/d/e/2PACX-1vQa672IV-XwWG9q-ujEJ0w72QCBzf4jVMNI-trwLuKt9kk_ewe0l6Mk6YSWoo9UYMETbJ5RZ3akKyux/pub?start=false&loop=false&delayms=3000)
-* [Getting Job Hunt Ready](https://docs.google.com/presentation/d/e/2PACX-1vTDlRAiXiR_PSsAFGQtP8je_pcUWmLdk1kYc4jO4hTyzxDuksNWUMdUITMNXp1pRFMM0gKDiAVyXL0c/pub?start=false&loop=false&delayms=3000)
+
+- [Overview](./pd_overview)
+- [Building Your Brand on LinkedIn](https://docs.google.com/presentation/d/e/2PACX-1vS8CZUjYkpnkTJrp2Ga8um-UUFUsJ-5JA85FF9x875J-l_eKy7IyL47sAt8kl_FOlg7rG5ntVxpk5he/pub?start=false&loop=false&delayms=3000)
+- [Networking, Outreach and Coffee Chats](./networking)
+- [Networking](https://docs.google.com/presentation/d/e/2PACX-1vQa672IV-XwWG9q-ujEJ0w72QCBzf4jVMNI-trwLuKt9kk_ewe0l6Mk6YSWoo9UYMETbJ5RZ3akKyux/pub?start=false&loop=false&delayms=3000)
+- [Getting Job Hunt Ready](https://docs.google.com/presentation/d/e/2PACX-1vTDlRAiXiR_PSsAFGQtP8je_pcUWmLdk1kYc4jO4hTyzxDuksNWUMdUITMNXp1pRFMM0gKDiAVyXL0c/pub?start=false&loop=false&delayms=3000)
 
 ## Additional Resources
-* [Chrome Dev Tools](./chrome_dev_tools)
-* [JSON for API development](../resources/json_for_api_development.md)
+
+- [Chrome Dev Tools](./chrome_dev_tools)
+- [JSON for API development](../resources/json_for_api_development)
+- [How the Web Works I: HTTP Request/Response Cycle](./how_the_web_works_http)
+- [How the Web Works II: APIs](./how_the_web_works_api)
```

Comment:
Such a small nitpick so feel free to disregard, but I feel like these 2 lessons shouldn't be in "additional resources" to me "additional resources" are nice to knows not need to knows, are things students should be exploring on their own time, etc. I wonder if we made a Web Development section header if these couldn't go in there?

Reply:
Yea, I looked through the available categories and couldn't decide where this lesson fit, so I stuck it here. Any suggestions are welcome.